src/components/common/ButtonlikeLink.vue

```diff
@@ -0,0 +1,41 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+});
+</script>
+
+<template>
+  <div class="buttonlike-link">
```

Comment:
While okay for now - I would definitely try to refactor this to a single "ButtonBase" component - this way we would only need to have the styles in one place and there would be no need to wonder which component to use where.

Reply:
I will definitely go for one versatile component :)